Clarification that you cannot use the existing GPO to deploy multiple… (#5045)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-application-control/deploy-multiple-windows-defender-application-control-policies.md

### Deploying policies

> [!NOTE]

> > You cannot use the "Deploy Windows Defender Application Control" group policy setting to deploy multiple CI policies. You must by any mean copy the *.cip files, both the baseline and the supplemental ones, to C:\Windows\System32\CodeIntegrity\CiPolicies\Active.

Reviewer: The idiom or phrase "by any means" is difficult to use properly outside of informal speech, unless used in a strong tone, indicating strict requirements bordering on a crisis. "By any means necessary" - kill someone or break the law if they try to stop you.
```suggestion
> You cannot use the "Deploy Windows Defender Application Control" group policy setting to deploy multiple CI policies. You will have to copy the `*.cip` files, both the baseline and the supplemental ones, to C:\Windows\System32\CodeIntegrity\CiPolicies\Active\.
```
- MarkDown uses asterisks to format text, so it can sometimes be necessary to use either an escape character (usually backslash) or back ticks (code fence) to mark the text containing the asterisk as monospaced code instead of keeping it as straight text.
- Added a backslash at the end, to make the ending period double as a directory location indicator.
